docs/plugin_development.md

```diff
@@ -68,14 +69,35 @@ The RosTopicDisplay is templated on the message type of this display:
 
 #### Providing Ogre media files
 
-It is possible to add your own meshes, scripts, etc. to RViz. 
+It is possible to add your own meshes, scripts, etc. to RViz.
 To make them available at runtime, use the cmake macro `register_rviz_ogre_media_exports` defined in `rviz_rendering`
 For instance, if you want to register a folder `test_folder/scripts` which resides in your project's source folder, write
 ```
 register_rviz_ogre_media_exports(DIRECTORIES "test_folder/scripts")
 ```
 *Note:* If you want to export folder hierarchies, each folder needs to be exported separately. Folders within exported folders are not automatically available.
 
+#### Writing a display which can only work with one transformation plugin
+
+Some of the default displays (e.g. TF display or LaserScan display) can only work with tf2 as transformation framework.
```

Comment:
The `laser_geometry` package used internally needs a `tf2_buffer`. Since it is not very easy to get around this (one would need to heavily rewrite some parts in `laser_geometry`), we decided that it's probably better to leave it out for now.